server/src/controllers/Events/resolver.ts

```diff
@@ -207,6 +212,11 @@ export class EventResolver {
       },
     });
 
+    const chapterUsers = await prisma.chapter_users.findMany({
+      where: { chapter_id: chapterId },
```

Comment:
```suggestion
      where: {
        chapter_id: chapterId,
        subscribed: true,
        chapter_role: { is: { name: 'administrator' } },
      },
```
Not too sure if this is worth changing, but after this, only chapter administrators would be passed to the `rsvpNotifyAdministrators`. This could slightly cut function, and `chapterUsers` variable could be renamed.

My mind was having hard time for a while, to connect why exactly `chapterUsers` are being passed to `rsvpNotifyAdministrators`.

Reply:
Thanks, @gikf, this approach makes much more sense.